.travis.yml

```diff
@@ -5,5 +5,8 @@ python:
   - "3.5"
 # command to install dependencies
 install: "pip install -r requirements.txt"
+before_script: 
+  - export PY_VERSION=`python -c 'import sys; print sys.version_info.major'`
+  - if [ $PY_VERSION == '2' ]; then ./lint.sh all; fi
```

Comment:
If this script is bash you should always perform tests in double brackets to prevent accidental interaction with globbing.

Also, always quote variable expansions to prevent accidental syntax errors if you get something unexpected.

```bash
if [[ "$PY_VERSION" == '2' ]]; then ./lint.sh all; fi
```